lms/djangoapps/completion_api/tests/test_serializers.py

```diff
@@ -91,14 +92,14 @@ def test_serialize_student_progress_object(self, serializer_cls, extra_body):
         expected = {
             'course_key': 'course-v1:abc+def+ghi',
             'completion': {
-                'earned': 16.0,
+                'earned': None,
                 'possible': 19.0,
-                'ratio': 16 / 19,
+                'ratio': None,
             },
         }
         expected.update(extra_body)
         self.assertEqual(
-            serial.data,
+            json.loads(json.dumps(serial.data)),
```

Comment:
There must be a better way to do this than to convert to JSON and back ;)

Reply:
@bradenmacdonald agreed, I was trying to fix the tests keeping in mind that this is a short-term solution. So to convert a nested OrderedDict to dict, this was the quick fix I came up with.

Reply:
Was this a problem?  I've never noticed an issue comparing equality between an OrderedDict and an unordered dict.  Or was it more about the readability of the diff?

Reply:
@jcdyer you are right. That conversion is not necessary. Will remove it.